indico/modules/events/templates/admin/event_keywords.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout/admin_page.html' %}
+{% from 'forms/_form.html' import simple_form %}
+
+{% block title %}
+    {% trans %}Allowed event keywords{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {% trans %}Define the list of allowed event keywords. An empty list implies any keyword is valid.
+    {% endtrans %}
```

Comment:
```suggestion
    {%- trans %
        }Define the list of allowed event keywords. An empty list implies any keyword is valid.
    {%- endtrans %}
```

just formatting cleanup. I think we should be clearer that this should only be used if one wants to limit the keywords that can be used.